pennylane/devices/default_mixed.py

```diff
@@ -271,9 +300,8 @@ def density_matrix(self, wires):
             array[complex]: complex array of shape ``(2 ** len(wires), 2 ** len(wires))``
             representing the reduced density matrix of the state prior to measurement.
         """
-        state = getattr(self, "state", None)
-        wires = self.map_wires(wires)
-        return qml.math.reduce_dm(state, indices=wires, c_dtype=self.C_DTYPE)
+        dim = 2**self.num_wires
```

Comment:
Why remove the ability to retrieve a reduced density matrix of a subsystem?

Reply:
You are right here it shouldn't be like this. I forgot this point. Thx for reminder!